daprdocs/content/en/operations/observability/metrics/metrics-overview.md

```diff
@@ -70,15 +70,178 @@ spec:
+
+
+#### High cardinality with path matching
+
+Configuration:
+```yaml
+http:
+  increasedCardinality: true
+  pathMatching:
+    - /orders/{orderID}
+```
+
+Metrics generated:
+```
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders/{orderID}",status="200"} 5
+```
+
+This example results from the same HTTP requests as the example above, but with path matching configured for the path `/orders/{orderID}`. By using path matching, we achieve reduced cardinality by grouping the metrics based on the matched path.
+
+#### High Cardinality without path matching
+
+Configuration:
+```yaml
+http:
+  increasedCardinality: true
+```
+
+Metrics generated:
+```
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders/1",status="200"} 1
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders/2",status="200"} 1
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders/3",status="200"} 1
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders/4",status="200"} 1
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders/5",status="200"} 1
+```
+
+For each request, a new metric is created with the request path. This process continues for every request made to a new order ID, resulting in unbounded cardinality since the IDs are ever-growing.
+
+
+### HTTP metrics exclude verbs
+
+The `excludeVerbs` option allows you to exclude specific HTTP verbs from being reported in the metrics. This can be useful in high-performance applications where memory savings are critical.
+
+### Examples of excluding HTTP verbs in metrics
+
+Here are some examples demonstrating how to exclude HTTP verbs in Dapr for managing HTTP metrics.
+
+#### Default - Include HTTP verbs
+
+Configuration:
+```yaml
+http:
+  excludeVerbs: false
+```
+
+Metrics generated:
+```
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders",status="200"} 1
+dapr_http_server_request_count{app_id="order-service",method="POST",path="/orders",status="200"} 1
+```
+
+In this example, the HTTP method is included in the metrics, resulting in a separate metric for each request to the `/orders` endpoint.
+
+#### Exclude HTTP verbs
+
+Configuration:
+```yaml
+http:
+  excludeVerbs: true
+```
+
+Metrics generated:
+```
+dapr_http_server_request_count{app_id="order-service",method="",path="/orders",status="200"} 2
+```
+
+In this example, the HTTP method is excluded from the metrics, resulting in a single metric for all requests to the `/orders` endpoint.
+
+### Configuring Custom Latency Buckets
+
+By default, Dapr employs predefined latency buckets for histogram metrics, listed below in milliseconds:
```

Comment:
Can you say what a latency bucket is? Is it the time period over which the metrics are gathered, so for the 1ms bucket it is gather all metrics every 1ms and the 1_000 bucket is gathering metrics every 1s (so it is less granular). Is that what they are?

Reply:
they are actually the bins in a [cumulative histogram](https://en.wikipedia.org/wiki/Histogram#Cumulative_histogram), that allows for fast quantile calculation.  The quantile is not exact (unless it lands at exactly at one of the bin edges) in the sense that the last bucket in your calculation you don't really know the exact point of each item in it but you can estimate.

So above it is very detail/precise between 1 and 10 ms, but less so on the upper values, but one could decide to have more detail at different ranges